### PR TITLE
feat(tooltip, popover): add functionality, which hide tooltip & popover after delay (#5572)

### LINKED ISSUE #5572: Hide tooltip after delay
**Is your feature request related to a problem? Please describe.**

Tooltip stays open if mouse is over

**Describe the solution you'd like**

Tooltip should disappear after some delay time

### PR DESCRIPTION
**Pull request by daniloff200**
closes #5572

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated tests.
 - [x] added/updated API documentation.
 - [x] added/updated demos.

### CODE REVIEW

File: demo/src/ng-api-doc.ts [hunk 1]
```diff
@@ -2715,7 +2715,13 @@ export const ngdoc: any = {
         "name": "delay",
         "defaultValue": "0",
         "type": "number",
-        "description": "<p>delay before showing the tooltip</p>\n"
+        "description": "<p>delay before showing the popover</p>\n"
```
Comment (IraErshova) [inline demo/src/ng-api-doc.ts:2941]: Did you add this code by thy script or manually?

File: src/popover/popover.directive.ts [hunk 2]
```diff
@@ -70,10 +70,15 @@ export class PopoverDirective implements OnInit, OnDestroy {
...
   /**
-   * Delay before showing the tooltip
+   * Delay before showing the popover
    */
   @Input() delay: number;
 
+  /**
+   * Delay before hiding the popover
+   */
+  @Input() hideAfterDelay: number;
```
Comment (IraErshova) [inline src/popover/popover.directive.ts]: I think new properties we should add via Config, shouldn't we?

File: src/tooltip/tooltip.directive.ts [hunk 3]
```diff
@@ -86,6 +86,11 @@ export class TooltipDirective implements OnInit, OnDestroy {
    */
   @Input() delay: number;
 
+  /**
+   * Delay before hiding the tooltip
+   */
+  @Input() hideAfterDelay: number;
```
Comment (IraErshova) [inline src/tooltip/tooltip.directive.ts]: the same about Config

**Comment by daniloff200** [comment]
hey, @dmitry-zhemchugov , I will need your help with fixing my e2e tests later :)

**Comment by dmitry-zhemchugov** [comment]
Need fix regarding cooldown of tooltip @daniloff200

**Review by dmitry-zhemchugov — CHANGES_REQUESTED** [review]
Need fix.

**Comment by daniloff200** [comment]
Project: ngx-bootstrap-demo 
 Url: https://ngx-bootstrap-demo--pr-5599-pccirowb.web.app 
 This link will expire at Thu 11 Mar 2021 07:40:19 AM GMT

**Comment by giovannidias1** [comment]
why this PR still open

**Pull request closed without merging**